packages/core/src/components/DotPagination/DotPagination.tsx

```diff
@@ -47,7 +45,7 @@ export interface HvDotPaginationProps
   /**
    * A Jss Object used to override or extend the styles applied.
    */
-  classes?: HvDotPaginationClasses;
+  classes?: Partial<HvDotPaginationClasses>;
```

Comment:
We no longer need `Partial` (nor above). It's included in `ExtractNames`.

Here it's triplicated!:

```tsx
classes?: Partial<Partial<Partial<ReturnType<typeof useClasses>["classes"]>>>;
```